modules/byo-vpc-single-az.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * assemblies/byo-vpc.adoc
+
+[id="byo-vpc-single-az_{context}"]
+= BYO VPC with a single availability zone required fields
```

Comment:
* Any way to start this (and the other module) with a gerund?
* Also, you're using "BYO" without ever have defining it. Maybe spell it out instead?

Same for elsewhere in the PR if you can also check